ios/AirGoogleMaps/AIRGoogleMapPolyline.m

```diff
@@ -29,11 +29,25 @@ -(void)setCoordinates:(NSArray<AIRMapCoordinate *> *)coordinates
   _coordinates = coordinates;
 
   GMSMutablePath *path = [GMSMutablePath path];
-  for(int i = 0; i < coordinates.count; i++)
+
+  if (!coordinates || coordinates.count == 0) 
   {
+    _polyline.map = nil; // Remove polyline from the map
```

Comment:
simply use removeAllCoordinates API instead

Reply:
Unfortunately it doesn't work. Also I've tried to assign nil to the path instead of map but without success. This is my code:

```
if (!coordinates || coordinates.count == 0) 
  {
    [path removeAllCoordinates]; // Doesn't work
    // path = nil // Doesn't work
    // _polyline.map = nil; // Remove polyline from the map
    return;
  }
 ```
 
 @salah-ghanim